GDJS/Runtime/cocos-renderers/runtimegame-cocos-renderer.js

```diff
@@ -111,3 +111,21 @@ gdjs.RuntimeGameCocosRenderer.prototype.openURL = function(url) {
 gdjs.RuntimeGameCocosRenderer.prototype.stopGame = function() {
     // TODO - Not implemented as not useful for most games on mobile and browsers
 }
+
+/**
+ * Get the canvas DOM element.
+ */
+gdjs.RuntimeGameCocosRenderer.prototype.getCanvas = function() {
+    return cc._canvas;
```

Comment:
I've not found anything about _canvas in the doc. Means that Cocos2d-JS don't want to expose it and us to use it, and this could break in future versions.
It's a risk to take I guess, in worst case it breaks screenshot taking but we handle this with at least an error message in the console.

Reply:
Found it. The canvas can be reached from `cc.game.canvas`